source/conf.py

```diff
@@ -21,8 +21,8 @@
 author = "NVIDIA"
 
 # Single modifiable version for all of the docs - easier for future updates
-stable_version = "24.06"
-nightly_version = "24.08"
+stable_version = "24.08"
+nightly_version = "24.10"
```

Comment:
Could we also update the `stable` section below to use CUDA 12.5 and Python 3.11?

For #377.

That's what we tested on the nightly version of the docs, in #402 .

https://github.com/rapidsai/deployment/blob/e03c1385efdc3540216e4f2d32e5c7afa085e29a/source/conf.py#L27-L42